shared/src/main/scala/org/jetbrains/sbt/structure/dataSerializers.scala

```diff
@@ -249,16 +271,17 @@ trait DataSerializers {
   implicit val dependencyDataSerializer: XmlSerializer[DependencyData] = new XmlSerializer[DependencyData] {
     override def serialize(what: DependencyData): Elem =
       <dependencies>
-        {what.projects.sortBy(_.project).map(_.serialize)}
+        {what.projects.serialize}
         {what.modules.sortBy(_.id.key).map(_.serialize)}
         {what.jars.sortBy(_.file).map(_.serialize)}
       </dependencies>
 
     override def deserialize(what: Node): Either[Throwable,DependencyData] = {
-      val projects = (what \ "project").deserialize[ProjectDependencyData]
       val modules = (what \ "module").deserialize[ModuleDependencyData]
       val jars = (what \ "jar").deserialize[JarDependencyData]
-      Right(DependencyData(projects, modules, jars))
+      (what \ "projects").deserializeOne[Dependencies[ProjectDependencyData]].right.map { projectDependencies =>
```

Comment:
(minor)
It worth leaving `(what \ "projects").deserializeOne[Dependencies[ProjectDependencyData]]` extracted to a `val` as it was before.
It will improve readability a little